app/views/providers/means/housing_benefits/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_with model: @form,
+              url: providers_legal_aid_application_means_housing_benefits_path(@legal_aid_application),
+              method: :patch do |f| %>
+  <%= page_template page_title: page_title, template: :basic, form: f do %>
+    <%= f.govuk_radio_buttons_fieldset :housing_benefit, legend: {tag: "h1", size: "xl"} do %>
+      <%= f.govuk_radio_button :housing_benefit, "true", link_errors: true do %>
+        <%= f.govuk_text_field :housing_benefit_amount, width: "one-quarter", prefix_text: "£" %>
+        <%= f.govuk_collection_radio_buttons :housing_benefit_frequency,
+                                             RegularTransaction::FREQUENCIES,
```

Comment:
Should/does this display all the regular transaction frequencies? On the page in Miro it only has every week, every two weeks and every four weeks as options

Reply:
I have just realised this branch doesn't include the updated frequencies as that branch hasn't been merged to main yet. Once these are combined might just need to add a helper method to restrict the frequencies to those mentioned on the miro board (assuming these are accurate)